frontend/src/pages/ActivityReport/Pages/Review/Approver/Review.js

```diff
@@ -25,13 +29,15 @@ const Review = ({
           <span className="text-bold">Creator notes</span>
           <br />
           <br />
-          { additionalNotes || 'No creator notes' }
+          <Editor readOnly toolbarHidden defaultEditorState={defaultEditorState} />
```

Comment:
Is `Editor` being used directly just for `readOnly` fields?

Reply:
Yeah, the `RichEditor` component is setup that it would only be used when expecting input and not read only. I can update the component to make it work  both ways or I can see if `HtmlReviewItem` would work here as well.

Reply:
Coming back to this, the `HtmlReviewItem` expects to be used in the Review page so I would have to extend it to support being used somewhere else. I don't think that would be the way to go when we are doing a simple one liner here.